gcconnex-kube.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: gcconnex-deployment
+  namespace: gctools-test
+spec:
+  replicas: 2 # tells deployment to run 2 pods matching the template
```

Comment:
How are sessions handled in GCconnex?  Are they persisted in the database?  Or are they php's default file based sessions?  If they are in the database all good; otherwise having multiple replcas will cause problems with authentication...  also if there are any other stateful requests  that need to work with temporary files, or anything else that is specific to a single host, then more than 1 replica will blow up.

Reply:
Should be database from what I remember... I'll double check that.